spec/requests/monthly_report_comment_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe MonthlyReportCommentsController, type: :request do
+  let(:user) { create(:user) }
+  let(:report) { create(:monthly_report) }
+
+  before { login user }
+
+  describe '#create POST /monthly_report_comments' do
+    let(:comment) { build(:monthly_report_comment, monthly_report: report) }
+    let(:comment_params) { { comment: comment.attributes.slice('comment', 'monthly_report_id') } }
```

Comment:
__[rubocop]__
* Line is too long. [98/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)